apps/cms/lib/repo.ex

```diff
@@ -182,6 +189,28 @@ defmodule CMS.Repo do
     end
   end
 
+  # BEGIN PAGE CACHING #
+
+  @behaviour Nebulex.Caching.KeyGenerator
+
+  @impl true
+  def generate(_, _, [path, %{}]) do
+    "/cms/#{String.trim(path, "/")}"
+  end
+
+  @impl true
+  def generate(_, _, [path, params]) do
+    "/cms/#{String.trim(path, "/")}" <> params_to_string(params)
+  end
```

Comment:
Creates a key like above, but also handles parameters so you can have something like `/cms/foo?bar=baz&bop=bam`